source/develop/release-management/features/virt/autoinstall-windows-drivers.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Automatic installation of Windows drivers
+category: feature
+authors: Tomáš Golembiovský
+feature_name: Automatic installation of Windows drivers
+feature_modules: engine, vdsm
+feature_status: In Progress
+---
+
+# Automatic installation of Windows drivers
+
+## Summary
+
+The feature simplifies the installation of MS Windows on a VM when virtio disk
+and network drivers are used. oVirt provides a floppy with file
```

Comment:
I thought we wanted to remove the use of the floppy? Does this work with a CDROM as well?